db/trie/mptrie/leafnode.go

```diff
@@ -55,7 +54,7 @@ func newLeafNodeFromProtoPb(pb *triepb.LeafPb, hashVal []byte) *leafNode {
 		value: pb.Value,
 	}
 	l.cacheNode.serializable = l
-	if err := logNode(_nodeTypeLeaf, _actionTypeNew, hashVal, l); err != nil {
+	if err := logNode(_nodeTypeLeaf, _actionTypeNew, l, nil); err != nil {
```

Comment:
not working here

Reply:
see the following code, first, it will check `hashVal`, only use cli if `hashVal` is not set.
https://github.com/iotexproject/iotex-core/blob/715ccb732d10210dc0013ffacaf63eddfdacf11e/db/trie/mptrie/cachenode.go#L25-L32